src/kernel/src/rtld/module.rs

```diff
@@ -16,12 +16,14 @@ use std::io::Write;
 use std::path::Path;
 use std::sync::Arc;
 
+pub type ModuleHandle = u32;
```

Comment:
Better to use tuple struct instead to achieve strong typing.

Reply:
Sure, no problem.